modules/oc-mirror-imageset-config-params.adoc

```diff
@@ -0,0 +1,212 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-disconnected.adoc
+
+[id="oc-mirror-imageset-config-params_{context}"]
+= Image set mirroring configuration parameters
+
+// TODO intro
+
+// TODO: Updated all of the "mappings" to "String" like elsewhere in our doc - okay?
```

Comment:
@afflom @jpower432 Just as an FYI, I updated all instances of "(mapping)" to stay "String", to be consistent with how we write it elsewhere in the docs. Like here for example: https://deploy-preview-40474--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations.html#installation-configuration-parameters-optional_installing-aws-customizations

I also removed the "Kind" row, because this table is specific to the `ImageSetConfiguration`